usaspending_api/agencies/management/commands/load_agencies.py

```diff
@@ -0,0 +1,270 @@
+import logging
+
+from collections import namedtuple
+from distutils.util import strtobool
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from pathlib import Path
+from psycopg2.extras import execute_values
+from psycopg2.sql import SQL
+from usaspending_api.common.csv_helpers import read_csv_file_as_list_of_dictionaries
+from usaspending_api.common.etl import ETLQueryFile, ETLTable, mixins
+from usaspending_api.common.helpers.sql_helpers import get_connection
+from usaspending_api.common.helpers.text_helpers import standardize_nullable_whitespace as prep
+from usaspending_api.common.helpers.timing_helpers import Timer
+from usaspending_api.references.constants import DOD_ARMED_FORCES_CGAC
+
+logger = logging.getLogger("console")
+
+Agency = namedtuple(
+    "Agency",
+    [
+        "row_number",
+        "cgac_agency_code",
+        "agency_name",
+        "agency_abbreviation",
+        "frec",
+        "frec_entity_description",
+        "frec_abbreviation",
+        "subtier_code",
+        "subtier_name",
+        "subtier_abbreviation",
+        "toptier_flag",
+        "is_frec",
+        "user_selectable",
+        "mission",
+        "website",
+        "congressional_justification",
+        "icon_filename",
+        "include_toptier_without_subtier",
+    ],
+)
+
+MAX_CHANGES = 75
+
+
+class Command(mixins.ETLMixin, BaseCommand):
+    help = (
+        "Loads CGACs, FRECs, Subtier Agencies, Toptier Agencies, and Agencies.  Load is all or nothing.  "
+        "If anything fails, nothing gets saved."
+    )
+
+    agency_file = None
+    force = False
+
+    etl_logger_function = logger.info
+    etl_dml_sql_directory = Path(__file__).resolve().parent / "load_agencies_sql"
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "agency_file",
+            metavar="AGENCY_FILE",
+            help="Path (for local files) or URI (for http(s) or S3 files) of the raw agency CSV file to be loaded.",
```

Comment:
No need to change the help text, just wanted to make the point that a file scheme URI works too
```file://....```